src/App.tsx

```diff
@@ -1,5 +1,25 @@
+import SearchHeader from "./components/header/SearchHeader";
+import ClothingCardsGrid from "./components/clothingGrid/ClothingCardsGrid";
+import Sidebar from "./components/sidebar/Sidebar";
+import { useState } from "react";
+
 function App() {
-  return <main></main>;
+  const [isSidebarOpen, setIsSidebarOpen] = useState(false);
+
+  const handleOpenSidebar = () => {
+    setIsSidebarOpen(!isSidebarOpen);
```

Comment:
When setting a state based on the previous value (e.g. a counter or a toggle), please use the provided value in the [setState hook](https://react.dev/reference/react/useState#updating-state-based-on-the-previous-state).
```suggestion
    setIsSidebarOpen(prev => !prev);
```